pagopa/io-app
# feat: [IOPID-142] Remove IntesaID and update fallback IdP list and FAQs

## Short description
This PR removes [IntesaID](https://www.agid.gov.it/it/agenzia/stampa-e-comunicazione/notizie/2023/03/08/dal-23-aprile-2023-intesa-non-sara-piu-gestore-spid) from the fallback IdP list and updates the corresponding fallback FAQs.

| Before | After |
| - | - |
| <img src="https://user-images.githubusercontent.com/16268789/231779196-29c6b8da-59df-49cc-9354-f9126a2b9e0d.png" width="150" /> | <img src="https://user-images.githubusercontent.com/16268789/231779279-6de3d01d-9b8e-4b8b-8275-de4603cd5caf.png" width="150" /> | 

## List of changes proposed in this pull request
- [ts/utils/idps.ts](https://github.com/pagopa/io-app/pull/4538/files#diff-b497c511ebcce245d32a97d61f147ce3bf39ff56a0f81c2d18f0372f1d231dbe): Updates the fallback IdP list according to the actual [remote one](https://github.com/pagopa/io-services-metadata/blob/master/spid/idps/list.json).
- Updates already existing IdP image logos and add the missing ones.
- [locales/en/faq/faq2.md](https://github.com/pagopa/io-app/pull/4538/files#diff-f37f4a885f214ce178190c11459eeb758497fc053dcb09be14a920ddf398f768)
- [locales/it/faq/faq2.md](https://github.com/pagopa/io-app/pull/4538/files#diff-47fdda432bd31e39ac31a763f8b9f22be178c46f5dde9b191004853f14eae75e): removes Intesa from the list and add all missing new IdPs.
- [locales/en/faq/faq3.md](https://github.com/pagopa/io-app/pull/4538/files#diff-86de343215580a033b60cfdf4e433b37976081c16ce6a15b5e0398d5bd5de453)
- [locales/it/faq/faq3.md](https://github.com/pagopa/io-app/pull/4538/files#diff-12b47cc1018a227a24dbea5a8e63915abe2a579d2b8d8cd64b24ff123538d9e0): removes Intesa from the list and aligns the markdown to the [remote one](https://github.com/pagopa/io-services-metadata/blob/master/contextualhelp/data.json).

## How to test
Install a fresh app pointing to production blocking this HTTP call (https://assets.cdn.io.pagopa.it/spid/idps/list.json). In the IdP list grid, you should NOT see the **IntesaID** button.
